schema/encoding/jsonschema/jsonschema_test.go

```diff
@@ -261,232 +205,8 @@ func TestObjectWithNilSchema(t *testing.T) {
-
-	_, err := isValidJSON(b.Bytes())
-	assert.NoError(t, err)
-}
-
-func TestObjectFieldWithRequired(t *testing.T) {
-	s := &schema.Schema{
-		Description: "A list of students",
-		Fields: schema.Fields{
-			"student": schema.Field{
-				Validator: &schema.Object{
-					Schema: &schema.Schema{
-						Fields: schema.Fields{
-							"name": schema.Field{
-								Description: "a student name",
-								Required:    true,
-								Default:     "Unknown",
-								Validator: &schema.String{
-									MinLen: 0,
-									MaxLen: 10,
-								},
-							},
-							"class": schema.Field{
-								Default: "Unassigned",
-								Validator: &schema.String{
-									MinLen: 0,
-									MaxLen: 10,
-								},
-							},
-						},
-					},
-				},
-			},
-		},
-	}
-
-	b := new(bytes.Buffer)
-	encoder := NewEncoder(b)
-	assert.NoError(t, encoder.Encode(s))
-
-	m, err := isValidJSON(b.Bytes())
-	assert.NoError(t, err)
-
-	a := assert.New(t)
-	a.Equal("object", m["type"])
-	a.Equal("A list of students", m["title"])
-	p, ok := m["properties"].(map[string]interface{})
-	a.True(ok)
-
-	a.NotNil(p["student"])
-	student, ok := p["student"].(map[string]interface{})
-	a.True(ok)
-
-	a.Equal("object", student["type"])
-	properties, ok := student["properties"].(map[string]interface{})
-	a.True(ok)
-
-	_, ok = properties["name"].(map[string]interface{})
-	a.True(ok)
-
-	a.Equal(copyStringToInterface([]string{"name"}), student["required"])
-}
-
-func TestArrayOfObjects(t *testing.T) {
-	s := &schema.Schema{
-		Description: "A list of students",
-		Fields: schema.Fields{
-			"students": schema.Field{
-				Validator: &schema.Array{
-					ValuesValidator: &schema.Object{
-						Schema: &schema.Schema{
-							Fields: schema.Fields{
-								"student": schema.Field{
-									Description: "a student",
-									Required:    true,
-									Default:     "Unknown",
-									Validator: &schema.String{
-										MinLen: 1,
-										MaxLen: 10,
-									},
-								},
-								"class": schema.Field{
-									Default: "Unassigned",
-									Validator: &schema.String{
-										MinLen: 0, // Default value.
-										MaxLen: 10,
-									},
-								},
-							},
-						},
-					},
-				},
-			},
-		},
-	}
-	const expect = `{
-		"type": "object",
-		"title": "A list of students",
-		"properties": {
-			"students": {
-				"type": "array",
-				"items": {
-					"type": "object",
-					"properties": {
-						"student": {
-							"type": "string",
-							"description": "a student",
-							"default": "Unknown",
-							"minLength": 1,
-							"maxLength": 10
-						},
-						"class": {
-							"type": "string",
-							"default": "Unassigned",
-							"maxLength": 10
-						}
-					},
-					"required": ["student"]
-				}
-			}
-		}
-	}`
-
-	b := new(bytes.Buffer)
-	encoder := NewEncoder(b)
-	assert.NoError(t, encoder.Encode(s))
-	assert.JSONEq(t, expect, b.String())
-}
-
-func TestDefaultEncodingWithStringFieldAndIntegerDefault(t *testing.T) {
```

Comment:
OK, haven't added any tests for default, so this was removed by mistake. Will add some tests to replace it.